src/Storages/MergeTree/MergeTreeIndexConditionBloomFilter.cpp

```diff
@@ -511,6 +511,8 @@ bool MergeTreeIndexConditionBloomFilter::traverseASTEquals(
     RPNElement & out,
     const ASTPtr & parent)
 {
+    std::cerr << "MergeTreeIndexConditionBloomFilter::traverseASTEquals " << function_name << " ast " << key_ast->formatForErrorMessage() << std::endl;
```

Comment:
Debug message left.

Reply:
I don't like to submit PRs with typos fixes, but I came up with a style-check for std::cerr/std::cout - #29464